idx/memory/memory.go

```diff
@@ -376,76 +376,161 @@ func (m *MemoryIdx) GetPath(orgId int, path string) []idx.Archive {
 	return archives
 }
 
-func (m *MemoryIdx) Tag(orgId int, tag string, from int64) map[string]uint32 {
+func (m *MemoryIdx) TagDetails(orgId int, key, filter string, from int64) (map[string]uint64, error) {
```

Comment:
the change from uint32 to uint64 is only useful if we expect any tagkey-tagval combo to occur more than 4 billion times right? fine by me, hope we hit that number sometime ;) important to note here that these counts are per-instance, but they get added up with all other instances, where we get larger aggregate counts.

Reply:
yes, better be future proof :) It also has the advantage that in `clusterTagDetails` no new map needs to be instantiated for the sums of all instance counts across the cluster, and I'd guess that when aggregating across a large cluster the 4 billion count is easier to reach than in a single MT.